settings.gradle

```diff
@@ -20,3 +20,6 @@ project(":frameworks/kafka").name = "kafka"
 
 include 'frameworks/hdfs'
 project(":frameworks/hdfs").name = "hdfs"
+
+include 'frameworks/template'
```

Comment:
The `new-service` script does indeed add new lines here when generating a new project.  Adding the template project here makes it get built and tested like all other projects, so we prove it's a valid project.  Otherwise it may not be kept up to date as a valid project which others should be based on.